models/Verification.py

```diff
@@ -0,0 +1,40 @@
+# Std Lib Imports
```

Comment:
what is this file for?

Reply:
this was a db model for the verification system I was working on, forgot it was in this PR.